operators/StartMultitenancyOperators.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
Because it is a start file specific to PG, please name it as StartPGMultitenancyOperator.js

Reply:
Renamed to StartPostgresqlMTOperators.js